content/en/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,119 @@
+<label><div class="col40"><strong>GitHub username</strong></div><div class="col60"><input type="text" pattern="^@?[a-zA-Z\d](?:[a-zA-Z\d]|-(?=[a-zA-Z\d])){0,38}$" name="github_user" class="form-element" required></label></div>
+<div class="clearfix" style="margin-bottom: 5px;"></div>
+<label><div class="col40"><strong>Email address</strong></div><div class="col60"><input type="email" name="email" class="form-element" required></label></div>
+<div class="clearfix"></div>
+<div class="form-group"><input type="checkbox" id="accept_terms" name="accept_terms" class="form-element" required><label for="accept_terms"><div style="float: left; width: 90%;">I have read the <a href="https://static.dacdn.de/docs/conditions-hacktoberfest-2021.pdf">conditions of participation and privacy policy</a> and accept them.<sup class="color-teal-700">*</sup></div></label></div>
+<div class="form-group"><input type="checkbox" id="accept_us_transfers" name="accept_us_transfers" class="form-element" required><label for="accept_us_transfers"><div style="float: left; width: 90%;">For my participation, I want to use the US platform GitHub (<a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-privacy-statement">privacy policy</a>), for which the organizer cannot guarantee the same level of data protection as in the EU. There is a risk that my data may be subject to access by US authorities, and I may not have effective legal remedies. I agree to this.<sup class="color-teal-700">*</sup></div></label></div>
+<input type="hidden" name="language" value="en">
+<input type="hidden" name="year" value="2021">
+<div style="float: right; margin-top: 10px;"><input class="button button-primary" type="submit" value="Register"></label></div>
+<div class="clearfix"></div>
+</form>
+</div>
+
+You can also participate without using GitHub. To do so, please email us at <hacktoberfest@datenanfragen.de>. The email has to indicate your desire to participate in the event and your agreement to the [conditions of participation and privacy policy](https://static.dacdn.de/docs/conditions-hacktoberfest-2021.pdf). It also has to contain at least one Git patch.
+
+By the way: If you _also_ want to participate in the official Hacktoberfest, you have to [register separately](https://hacktoberfest.digitalocean.com/). That is however **not** necessary to participate in our event.
+
+<a id="repos"></a>
+
+## Which repositories qualify for the event?
+
+For the event, we count pull requests and patches to following repositories as valid contributions:
+
+* <https://github.com/datenanfragen/website>
+* <https://github.com/datenanfragen/data>
+* <https://github.com/datenanfragen/backend>
+* <https://github.com/datenanfragen/company-json-generator>
+* <https://github.com/datenanfragen/letter-generator>
+* <https://github.com/datenanfragen/media>
+* <https://github.com/datenanfragen/locate-contacts-addon>
+* <https://github.com/zner0L/postcss-fonticons>
+
```

Comment:
Let's also add https://github.com/datenanfragen/data-imports this year. It's probably not likely that anyone will contribute there but it would be nice.